### DISCUSSION TITLE
Dex support for fungible_asset standard.

### DISCUSSION THREAD

**Discussion by ctb0k33** [aptos-labs/aptos-developer-discussions#289] [Questions] [answered]
### Discord user ID

ctb0k33

### Describe your question in detail.

Hello everyone, I just read the example bonding_curve_launchpad in aptos move example. However, when reading that, i saw that the example use a swap module in the testnet. I have a question, if i want to create a pair on popular DEXs like PancakeSwap or LiquidSwap, do they support fungible_asset standard yet ?

### Which operating system are you using?

Windows

### Which SDK or tool are you using? (if any)

TypeScript SDK

**Comment by kshitijc1506** [comment, marked as answer]
Currently, FA is not supported by either Liquidswap (should be in progress AFAIK) or Pancakeswap. DEXs that already support FA are [Panora](https://panora.exchange/) (aggregator) and [Cellana](https://cellana.finance/).

**Reply by Alexzander-Stone** [reply]
For just a little more info @ctb0k33, you can see details on the transition from Coin to Fungible Asset Standard across the ecosystem [here](https://aptos.dev/en/build/smart-contracts/fungible-asset#migration-from-coin-to-the-fungible-asset-standard).
Although smart contracts won't need to be modified, off-chain interactions will.

**Reply by ctb0k33** [reply]
thanks for information !

**Reply by 0xbe1** [reply]
@kshitjee Panora is a dex aggregator, not a dex though
